File: msa/web-service/src/main/java/com/example/spring/webservice/controller/BoardApiController.java
package com.example.spring.webservice.controller;

import lombok.RequiredArgsConstructor;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequiredArgsConstructor
@RequestMapping("/api/boards")
public class BoardApiController {

    @GetMapping("/search")
    public void searchBoards(){

    }

}

File: kotlin/kotlin-part-4/src/main/kotlin/com/example/spring/kotlinpart4/controller/BoardApiController.kt
package com.example.spring.kotlinpart4.controller

import com.example.spring.kotlinpart4.service.BoardService
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/api/boards")
class BoardApiController(private val boardService: BoardService) {


}